src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+function promise1() {
+  return new Promise((resolve, reject) => {
+    const logo = document.getElementById('logo');
+
+    logo.addEventListener('click', () => {
+      const successDiv = document.createElement('div');
+
+      successDiv.classList.add('message');
+      successDiv.textContent = 'Promise was resolved!';
+      document.body.appendChild(successDiv);
+
+      resolve();
+    });
+  });
+}
+
+function promise2() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const errorDiv = document.createElement('div');
+
+      errorDiv.classList.add('message');
+      errorDiv.classList.add('error-message');
+      errorDiv.textContent = 'Promise was rejected!';
+      document.body.appendChild(errorDiv);
+
+      reject(new Error('Promise was rejected!'));
+    }, 3000);
+  });
+}
+
```

Comment:
You should create two function which will be call on resolve and reject status with created HTML elements, they can be, e.g:

```
const onSuccess = () => {
  document.body.insertAdjacentHTML('beforeend',
    `<div class="message">Promise was resolved!</div>`
  );
};

const onError = () => {
  document.body.insertAdjacentHTML('beforeend',
    `<div class="message error-message">Promise was rejected!</div>`
  );
};
```